const interpretation1 = [
  {
    id: "1",
    Interpretation:
      "A variant with accession number VCV000225241 has been identified. This variant is titled 'NM_001163435.3(TBCK):c.1370del (p.Asn457fs)'.\nIt is a Deletion variant, with a cDNA change of c.1370del and a protein change of N394fs, N418fs, N285fs, N457fs.\nThe variant is located on chromosome 4, spanning from position 106235348 to 106235348.\nThis variant is associated with the trait 'Hypotonia, infantile, with psychomotor retardation and characteristic facies 3' and has been classified as 'Pathogenic'.\nThe review status for this variant is 'criteria provided, multiple submitters, no conflicts', and it was last evaluated on 2025/01/13 00:00.\ndbSNP link: https://www.ncbi.nlm.nih.gov/snp/746860249\nClinGen link: https://reg.clinicalgenome.org/redmine/projects/registry/genboree_registry/by_caid?caid=CA358417\nTBCK, also known as TBC1 domain-containing kinase, plays a crucial role in regulating cellular processes such as endocytosis, exocytosis, and membrane trafficking. A pathogenic variant, NM_001163435.3(TBCK):c.1370del (p.Asn457fs), has been identified, leading to hypotonia, psychomotor retardation, and characteristic facial features in affected individuals. This deletion variant disrupts the protein's function, resulting in severe developmental delays and neurological impairments.\nHypotonia, infantile, with psychomotor retardation and characteristic facies 3\nDescription: A rare genetic disorder characterized by low muscle tone (hypotonia), developmental delay, intellectual disability, and distinctive facial features. The condition is usually diagnosed at birth or in early infancy and can lead to feeding difficulties, respiratory problems, and delayed motor milestones. The disease has a variable severity of symptoms and may be associated with other complications such as seizures and hearing loss.\nSabanathan S, Gulhane D, Mankad K et al.. Expanding the phenotype of children presenting with hypoventilation with biallelic TBCK pathogenic variants and literature review. Neuromuscular disorders : NMD. 2023 Jan. PMID: 36522252\nAdam MP, Feldman J, Mirzaa GM et al.. TBCK-Related Neurodevelopmental Disorder. . 1993. PMID: 40504974\nWu J, Lu G. Multiple functions of TBCK protein in neurodevelopment disorders and tumors. Oncology letters. 2021 Jan. PMID: 33240423\nZapata-Aldana E, Kim DD, Remtulla S et al.. Further delineation of TBCK - Infantile hypotonia with psychomotor retardation and characteristic facies type 3. European journal of medical genetics. 2019 Apr. PMID: 30103036\nRiffe RM, Downes GB. Neurogenetic disorders associated with mutations in the FERRY complex: a novel disease class? Biology open. 2025 Mar 15. PMID: 40062705",

    "Gene Name": "TBCK",
    "Protein Change": "p.N457Tfs*15",
    "Coding Change": "c.1370delA",
    "ClinVar Significance": "Pathogenic",
    "Genomiki Variant Classification": "Pathogenic",
    "Disease Associated":
      "Hypotonia, infantile, with psychomotor retardation and characteristic facies 3\n",
    Zygosity: "Homozygous",
    Chromosome: "chr4",
    Position: "107156504",
    "Ref Allele": "GT",
    "Alt Allele": "G",
    "Transcript ID": "NM_001163435",
    "Exon Number": "exon15",
    Depth: "44",
    "gnomAD Allelic Frequency": ".",
    "Gene Effect": "exonic",
    "Mutation Effect": "frameshift_deletion",
    SIFT: ".",
    Polyphen2: ".",
    MutationTaster: ".",
    dbSNPID: "rs746860249",
    "Inheritance Pattern": "Autosomal Recessive",
    "Evidence Code": "PM2,PP5,PVS1",
    OMIM: "616900",
  },
  {
    id: "2",

    Interpretation:
      "A variant with accession number VCV000001900 has been identified. This variant is titled 'NM_001370658.1(BTD):c.1270G>C (p.Asp424His)'.\nIt is a single nucleotide variant variant, with a cDNA change of c.1270G>C and a protein change of D424H.\nThe variant is located on chromosome 3, spanning from position 15645186 to 15645186.\nThis variant is associated with the trait 'not provided' and has been classified as 'Conflicting classifications of pathogenicity'.\nThe review status for this variant is 'criteria provided, conflicting classifications', and it was last evaluated on 2025/03/01 00:00.\ndbSNP link: https://www.ncbi.nlm.nih.gov/snp/13078881\nClinGen link: https://reg.clinicalgenome.org/redmine/projects/registry/genboree_registry/by_caid?caid=CA090886\nThe BTD gene plays a crucial role in the breakdown of certain amino acids, particularly phenylalanine. A variant, NM_001370658.1(BTD):c.1270G>C (p.Asp424His), has been identified on chromosome 3. This single nucleotide change may impact protein function, but its pathogenicity is disputed, with conflicting classifications. Further evaluation is needed to determine the variant's clinical significance and potential health effects.\nBiotinidase Deficiency\nDescription: Biotinidase deficiency is a rare genetic disorder caused by the inability to recycle biotin, a vitamin necessary for energy production and maintenance of healthy skin, hair, and mucous membranes. This condition leads to a buildup of biotin in the bloodstream, which eventually depletes due to impaired recycling mechanisms. Symptoms include neurological issues such as seizures, developmental delays, muscle weakness, and dermatitis. If left untreated, severe cases can lead to life-threatening complications, including coma or death. Early detection through newborn screening is crucial for effective management with biotin supplements.\nYeo CJJ, Tizzano EF, Darras BT. Challenges and opportunities in spinal muscular atrophy therapeutics. The Lancet. Neurology. 2024 Feb. PMID: 38267192\nAdam MP, Feldman J, Mirzaa GM et al.. Biotinidase Deficiency. . 1993. PMID: 20301497\nDodsworth BT, Reeve K, Falco L et al.. Development and validation of an international preoperative risk assessment model for postoperative delirium. Age and ageing. 2023 Jun 1. PMID: 37290122\nSadeghirad B, Dodsworth BT, Schmutz Gelsomino N et al.. Perioperative Factors Associated With Postoperative Delirium in Patients Undergoing Noncardiac Surgery: An Individual Patient Data Meta-Analysis. JAMA network open. 2023 Oct 2. PMID: 37819663",
    "Gene Name": "BTD",
    "Protein Change": "p.D424H",
    "Coding Change": "c.G1270C",
    "ClinVar Significance": "Pathogenic/Likely_pathogenic",
    "Genomiki Variant Classification": "Uncertain Significance",
    "Disease Associated": "Biotinidase deficiency\n",
    Zygosity: "Heterozygous",
    Chromosome: "chr3",
    Position: "15686693",
    "Ref Allele": "G",
    "Alt Allele": "C",
    "Transcript ID": "NM_001370658",
    "Exon Number": "exon4",
    Depth: "36",
    "gnomAD Allelic Frequency": "0.0285",
    "Gene Effect": "exonic",
    "Mutation Effect": "nonsynonymous_SNV",
    SIFT: "D",
    Polyphen2: "D",
    MutationTaster: "A",
    dbSNPID: "rs13078881",
    "Inheritance Pattern": "Autosomal Recessive",
    "Evidence Code": "PP5,PM5,PM1,PP3,PP2,BS2,BA1",
    OMIM: "253260",
  },
];
const interpretation2 = [
  {
    id: "1",
    Interpretation:
      "A variant with accession number VCV000015417 has been identified. This variant is titled 'NM_000518.5(HBB):c.126_129del (p.Phe42fs)'.\nIt is a Deletion variant, with a cDNA change of c.126_129del and a protein change of F42fs.\nThe variant is located on chromosome 11, spanning from position 5226763 to 5226766.\nThis variant is associated with the trait 'Beta-thalassemia HBB/LCRB' and has been classified as 'Pathogenic/Likely pathogenic'.\nThe review status for this variant is 'criteria provided, multiple submitters, no conflicts', and it was last evaluated on 2024/11/19 00:00.\ndbSNP link: https://www.ncbi.nlm.nih.gov/snp/80356821\nClinGen link: https://reg.clinicalgenome.org/redmine/projects/registry/genboree_registry/by_caid?caid=CA125284\nThe HBB gene encodes hemoglobin subunit beta, crucial for oxygen transport in red blood cells. A deletion variant (VCV000015417) affects codons 126-129, leading to a premature stop codon at position 42 (p.Phe42fs). This pathogenic/likely pathogenic variant is associated with Beta-thalassemia HBB/LCRB and can impair hemoglobin production, potentially causing severe anemia.\nSickle Cell Disease\n\nDescription: Sickle cell disease (SCD) is a group of inherited disorders that affect hemoglobin production in red blood cells. It occurs when an individual inherits two copies of the abnormal HBB* allele, one from each parent. This mutation causes the normally flexible and disk-shaped red blood cells to become rigid and take on a sickle or crescent shape under certain conditions, such as stress, fatigue, or cold temperatures. As these deformed cells move through small blood vessels, they can get stuck and cause blockages that lead to tissue damage, pain crises, anemia, infections, and increased risk of complications like acute chest syndrome and stroke.\nKato GJ, Piel FB, Reid CD et al.. Sickle cell disease. Nature reviews. Disease primers. 2018 Mar 15. PMID: 29542687\nPark SH, Bao G. CRISPR/Cas9 gene editing for curing sickle cell disease. Transfusion and apheresis science : official journal of the World Apheresis Association : official journal of the European Society for Haemapheresis. 2021 Feb. PMID: 33455878\nCao A, Galanello R. Beta-thalassemia. Genetics in medicine : official journal of the American College of Medical Genetics. 2010 Feb. PMID: 20098328\nEsoh K, Wonkam A. Evolutionary history of sickle-cell mutation: implications for global genetic medicine. Human molecular genetics. 2021 Apr 26. PMID: 33461216Harteveld CL, Achour A, Arkesteijn SJG et al.. The hemoglobinopathies, molecular disease mechanisms and diagnostics. International journal of laboratory hematology. 2022 Sep. PMID: 36074711",

    "Gene Name": "HBB",
    "Protein Change": "p.F42Lfs*19",
    "Coding Change": "c.126_129del",
    "ClinVar Significance": "Pathogenic/Likely_pathogenic",
    "Genomiki Variant Classification": "Pathogenic",
    "Disease Associated": "Sickle cell disease",
    Zygosity: "Heterozygous",
    Chromosome: "chr11",
    Position: "5247992",
    "Ref Allele": "CAAAG",
    "Alt Allele": "C",
    "Transcript ID": "NM_000518",
    "Exon Number": "exon2",
    Depth: "109",
    "gnomAD Allelic Frequency": "9.56E-05",
    "Gene Effect": "exonic",
    "Mutation Effect": "frameshift_deletion",
    SIFT: ".",
    Polyphen2: ".",
    MutationTaster: ".",
    dbSNPID: "rs80356821",
    "Inheritance Pattern": "Autosomal Recessive",
    "Evidence Code": "PM2,PP5,PVS1",
    OMIM: "603903",
  },
  {
    id: "2",
    Interpretation:
      "A variant with accession number VCV001702670 has been identified. This variant is titled 'NM_019109.5(ALG1):c.295C>T (p.Arg99Ter)'.\nIt is a single nucleotide variant variant, with a cDNA change of c.295C>T and a protein change of R99*.\nThe variant is located on chromosome 16, spanning from position 5073161 to 5073161.\nThis variant is associated with the trait 'ALG1-congenital disorder of glycosylation' and has been classified as 'Pathogenic/Likely pathogenic'.\nThe review status for this variant is 'criteria provided, multiple submitters, no conflicts', and it was last evaluated on 2024/10/17 00:00.\ndbSNP link: https://www.ncbi.nlm.nih.gov/snp/750451819\nClinGen link: https://reg.clinicalgenome.org/redmine/projects/registry/genboree_registry/by_caid?caid=CA7889778\nThe ALG1 gene plays a crucial role in glycosylation, a process essential for protein function. A pathogenic variant, NM_019109.5(ALG1):c.295C>T (p.Arg99Ter), has been identified on chromosome 16. This single nucleotide change leads to premature termination of the ALG1 protein, potentially causing an ALG1-congenital disorder of glycosylation. The variant is classified as pathogenic/likely pathogenic, highlighting its significant impact on gene function and associated clinical implications.\nCongenital disorder of glycosylation, type Ik\nDescription: A rare congenital disorder characterized by impaired protein glycosylation due to mutations in the ALG9 gene. This leads to a wide range of systemic and neurological symptoms, including developmental delay, muscle weakness, and seizures. The disorder is caused by the absence or significant reduction of dolichol-linked oligosaccharide formation, which is crucial for proper protein folding and function.\nXue Y, Zhao Y, Wu B et al.. A novel variant in ALG1 gene associated with congenital disorder of glycosylation: A case report and short literature review. Molecular genetics & genomic medicine. 2023 Aug. PMID: 37204045\nAdam MP, Feldman J, Mirzaa GM et al.. Congenital Disorders of N-Linked Glycosylation and Multiple Pathway Overview. . 1993. PMID: 20301507\nMakhamreh MM, Cottingham N, Ferreira CR et al.. Nonimmune hydrops fetalis and congenital disorders of glycosylation: A systematic literature review. Journal of inherited metabolic disease. 2020 Mar. PMID: 31420886\nMarques-da-Silva D, Dos Reis Ferreira V, Monticelli M et al.. Liver involvement in congenital disorders of glycosylation (CDG). A systematic review of the literature. Journal of inherited metabolic disease. 2017 Mar. PMID: 28108845",

    "Gene Name": "ALG1",
    "Protein Change": "p.R99X",
    "Coding Change": "c.C295T",
    "ClinVar Significance": "Pathogenic/Likely_pathogenic",
    "Genomiki Variant Classification": "Pathogenic",
    "Disease Associated": "Congenital disorder of glycosylation, type Ik",
    Zygosity: "Heterozygous",
    Chromosome: "chr16",
    Position: "5123162",
    "Ref Allele": "C",
    "Alt Allele": "T",
    "Transcript ID": "NM_019109",
    "Exon Number": "exon3",
    Depth: "272",
    "gnomAD Allelic Frequency": "3.19E-05",
    "Gene Effect": "exonic",
    "Mutation Effect": "stopgain",
    SIFT: ".",
    Polyphen2: ".",
    MutationTaster: "A",
    dbSNPID: "rs750451819",
    "Inheritance Pattern": "Autosomal Recessive",
    "Evidence Code": "PM2,PP5,PVS1",
    OMIM: "608540",
  },
];
const interpretation4 = [
  {
    id: "1",
    "Gene Name": "HBB",
    "Protein Change": "p.F42Lfs*19",
    "Coding Change": "c.126_129del",
    "ClinVar Significance": "Pathogenic/Likely_pathogenic",
    "Genomiki Variant Classification": "Pathogenic",
    "Disease Associated": "Sickle cell disease",
    Zygosity: "Heterozygous",
    Chromosome: "chr11",
    Position: "5247992",
    "Ref Allele": "CAAAG",
    "Alt Allele": "C",
    "Transcript ID": "NM_000518",
    "Exon Number": "exon2",
    Depth: "109",
    "gnomAD Allelic Frequency": "9.56E-05",
    "Gene Effect": "exonic",
    "Mutation Effect": "frameshift_deletion",
    SIFT: ".",
    Polyphen2: ".",
    MutationTaster: ".",
    dbSNPID: "rs80356821",
    "Inheritance Pattern": "Autosomal Recessive",
    "Evidence Code": "PM2,PP5,PVS1",
    OMIM: "603903",
    Interpretation:
      "A variant with accession number VCV000015417 has been identified. This variant is titled 'NM_000518.5(HBB):c.126_129del (p.Phe42fs)'.\nIt is a Deletion variant, with a cDNA change of c.126_129del and a protein change of F42fs.\nThe variant is located on chromosome 11, spanning from position 5226763 to 5226766.\nThis variant is associated with the trait 'Beta-thalassemia HBB/LCRB' and has been classified as 'Pathogenic/Likely pathogenic'.\nThe review status for this variant is 'criteria provided, multiple submitters, no conflicts', and it was last evaluated on 2024/11/19 00:00.\ndbSNP link: https://www.ncbi.nlm.nih.gov/snp/80356821\nClinGen link: https://reg.clinicalgenome.org/redmine/projects/registry/genboree_registry/by_caid?caid=CA125284\nThe HBB gene encodes hemoglobin subunit beta, crucial for oxygen transport in red blood cells. A deletion variant (VCV000015417) affects codons 126-129, leading to a premature stop codon at position 42 (p.Phe42fs). This pathogenic/likely pathogenic variant is associated with Beta-thalassemia HBB/LCRB and can impair hemoglobin production, potentially causing severe anemia.\nSickle Cell Disease\n\nDescription: Sickle cell disease (SCD) is a group of inherited disorders that affect hemoglobin production in red blood cells. It occurs when an individual inherits two copies of the abnormal HBB* allele, one from each parent. This mutation causes the normally flexible and disk-shaped red blood cells to become rigid and take on a sickle or crescent shape under certain conditions, such as stress, fatigue, or cold temperatures. As these deformed cells move through small blood vessels, they can get stuck and cause blockages that lead to tissue damage, pain crises, anemia, infections, and increased risk of complications like acute chest syndrome and stroke.\nKato GJ, Piel FB, Reid CD et al.. Sickle cell disease. Nature reviews. Disease primers. 2018 Mar 15. PMID: 29542687\nPark SH, Bao G. CRISPR/Cas9 gene editing for curing sickle cell disease. Transfusion and apheresis science : official journal of the World Apheresis Association : official journal of the European Society for Haemapheresis. 2021 Feb. PMID: 33455878\nCao A, Galanello R. Beta-thalassemia. Genetics in medicine : official journal of the American College of Medical Genetics. 2010 Feb. PMID: 20098328\nEsoh K, Wonkam A. Evolutionary history of sickle-cell mutation: implications for global genetic medicine. Human molecular genetics. 2021 Apr 26. PMID: 33461216Harteveld CL, Achour A, Arkesteijn SJG et al.. The hemoglobinopathies, molecular disease mechanisms and diagnostics. International journal of laboratory hematology. 2022 Sep. PMID: 36074711",
  },
  {
    id: "2",
    "Gene Name": "ALG1",
    "Protein Change": "p.R99X",
    "Coding Change": "c.C295T",
    "ClinVar Significance": "Pathogenic/Likely_pathogenic",
    "Genomiki Variant Classification": "Pathogenic",
    "Disease Associated": "Congenital disorder of glycosylation, type Ik",
    Zygosity: "Heterozygous",
    Chromosome: "chr16",
    Position: "5123162",
    "Ref Allele": "C",
    "Alt Allele": "T",
    "Transcript ID": "NM_019109",
    "Exon Number": "exon3",
    Depth: "272",
    "gnomAD Allelic Frequency": "3.19E-05",
    "Gene Effect": "exonic",
    "Mutation Effect": "stopgain",
    SIFT: ".",
    Polyphen2: ".",
    MutationTaster: "A",
    dbSNPID: "rs750451819",
    "Inheritance Pattern": "Autosomal Recessive",
    "Evidence Code": "PM2,PP5,PVS1",
    OMIM: "608540",
    Interpretation:
      "A variant with accession number VCV001702670 has been identified. This variant is titled 'NM_019109.5(ALG1):c.295C>T (p.Arg99Ter)'.\nIt is a single nucleotide variant variant, with a cDNA change of c.295C>T and a protein change of R99*.\nThe variant is located on chromosome 16, spanning from position 5073161 to 5073161.\nThis variant is associated with the trait 'ALG1-congenital disorder of glycosylation' and has been classified as 'Pathogenic/Likely pathogenic'.\nThe review status for this variant is 'criteria provided, multiple submitters, no conflicts', and it was last evaluated on 2024/10/17 00:00.\ndbSNP link: https://www.ncbi.nlm.nih.gov/snp/750451819\nClinGen link: https://reg.clinicalgenome.org/redmine/projects/registry/genboree_registry/by_caid?caid=CA7889778\nThe ALG1 gene plays a crucial role in glycosylation, a process essential for protein function. A pathogenic variant, NM_019109.5(ALG1):c.295C>T (p.Arg99Ter), has been identified on chromosome 16. This single nucleotide change leads to premature termination of the ALG1 protein, potentially causing an ALG1-congenital disorder of glycosylation. The variant is classified as pathogenic/likely pathogenic, highlighting its significant impact on gene function and associated clinical implications.\nCongenital disorder of glycosylation, type Ik\nDescription: A rare congenital disorder characterized by impaired protein glycosylation due to mutations in the ALG9 gene. This leads to a wide range of systemic and neurological symptoms, including developmental delay, muscle weakness, and seizures. The disorder is caused by the absence or significant reduction of dolichol-linked oligosaccharide formation, which is crucial for proper protein folding and function.\nXue Y, Zhao Y, Wu B et al.. A novel variant in ALG1 gene associated with congenital disorder of glycosylation: A case report and short literature review. Molecular genetics & genomic medicine. 2023 Aug. PMID: 37204045\nAdam MP, Feldman J, Mirzaa GM et al.. Congenital Disorders of N-Linked Glycosylation and Multiple Pathway Overview. . 1993. PMID: 20301507\nMakhamreh MM, Cottingham N, Ferreira CR et al.. Nonimmune hydrops fetalis and congenital disorders of glycosylation: A systematic literature review. Journal of inherited metabolic disease. 2020 Mar. PMID: 31420886\nMarques-da-Silva D, Dos Reis Ferreira V, Monticelli M et al.. Liver involvement in congenital disorders of glycosylation (CDG). A systematic review of the literature. Journal of inherited metabolic disease. 2017 Mar. PMID: 28108845",
  },
];
const interpretation3 = [
  {
    id: "1",
    Interpretation:
      "A variant with accession number VCV000225241 has been identified. This variant is titled 'NM_001163435.3(TBCK):c.1370del (p.Asn457fs)'.\nIt is a Deletion variant, with a cDNA change of c.1370del and a protein change of N394fs, N418fs, N285fs, N457fs.\nThe variant is located on chromosome 4, spanning from position 106235348 to 106235348.\nThis variant is associated with the trait 'Hypotonia, infantile, with psychomotor retardation and characteristic facies 3' and has been classified as 'Pathogenic'.\nThe review status for this variant is 'criteria provided, multiple submitters, no conflicts', and it was last evaluated on 2025/01/13 00:00.\ndbSNP link: https://www.ncbi.nlm.nih.gov/snp/746860249\nClinGen link: https://reg.clinicalgenome.org/redmine/projects/registry/genboree_registry/by_caid?caid=CA358417\nTBCK, also known as TBC1 domain-containing kinase, plays a crucial role in regulating cellular processes such as endocytosis, exocytosis, and membrane trafficking. A pathogenic variant, NM_001163435.3(TBCK):c.1370del (p.Asn457fs), has been identified, leading to hypotonia, psychomotor retardation, and characteristic facial features in affected individuals. This deletion variant disrupts the protein's function, resulting in severe developmental delays and neurological impairments.\nHypotonia, infantile, with psychomotor retardation and characteristic facies 3\nDescription: A rare genetic disorder characterized by low muscle tone (hypotonia), developmental delay, intellectual disability, and distinctive facial features. The condition is usually diagnosed at birth or in early infancy and can lead to feeding difficulties, respiratory problems, and delayed motor milestones. The disease has a variable severity of symptoms and may be associated with other complications such as seizures and hearing loss.\nSabanathan S, Gulhane D, Mankad K et al.. Expanding the phenotype of children presenting with hypoventilation with biallelic TBCK pathogenic variants and literature review. Neuromuscular disorders : NMD. 2023 Jan. PMID: 36522252\nAdam MP, Feldman J, Mirzaa GM et al.. TBCK-Related Neurodevelopmental Disorder. . 1993. PMID: 40504974\nWu J, Lu G. Multiple functions of TBCK protein in neurodevelopment disorders and tumors. Oncology letters. 2021 Jan. PMID: 33240423\nZapata-Aldana E, Kim DD, Remtulla S et al.. Further delineation of TBCK - Infantile hypotonia with psychomotor retardation and characteristic facies type 3. European journal of medical genetics. 2019 Apr. PMID: 30103036\nRiffe RM, Downes GB. Neurogenetic disorders associated with mutations in the FERRY complex: a novel disease class? Biology open. 2025 Mar 15. PMID: 40062705",
    "Gene Name": "TBCK",
    "Protein Change": "p.N457Tfs*15",
    "Coding Change": "c.1370delA",
    "ClinVar Significance": "Pathogenic",
    "Genomiki Variant Classification": "Pathogenic",
    "Disease Associated":
      "Hypotonia, infantile, with psychomotor retardation and characteristic facies 3\n",
    Zygosity: "Homozygous",
    Chromosome: "chr4",
    Position: "107156504",
    "Ref Allele": "GT",
    "Alt Allele": "G",
    "Transcript ID": "NM_001163435",
    "Exon Number": "exon15",
    Depth: "44",
    "gnomAD Allelic Frequency": ".",
    "Gene Effect": "exonic",
    "Mutation Effect": "frameshift_deletion",
    SIFT: ".",
    Polyphen2: ".",
    MutationTaster: ".",
    dbSNPID: "rs746860249",
    "Inheritance Pattern": "Autosomal Recessive",
    "Evidence Code": "PM2,PP5,PVS1",
    OMIM: "616900",
  },
  {
    id: "2",
    Interpretation:
      "A variant with accession number VCV000001900 has been identified. This variant is titled 'NM_001370658.1(BTD):c.1270G>C (p.Asp424His)'.\nIt is a single nucleotide variant variant, with a cDNA change of c.1270G>C and a protein change of D424H.\nThe variant is located on chromosome 3, spanning from position 15645186 to 15645186.\nThis variant is associated with the trait 'not provided' and has been classified as 'Conflicting classifications of pathogenicity'.\nThe review status for this variant is 'criteria provided, conflicting classifications', and it was last evaluated on 2025/03/01 00:00.\ndbSNP link: https://www.ncbi.nlm.nih.gov/snp/13078881\nClinGen link: https://reg.clinicalgenome.org/redmine/projects/registry/genboree_registry/by_caid?caid=CA090886\nThe BTD gene plays a crucial role in the breakdown of certain amino acids, particularly phenylalanine. A variant, NM_001370658.1(BTD):c.1270G>C (p.Asp424His), has been identified on chromosome 3. This single nucleotide change may impact protein function, but its pathogenicity is disputed, with conflicting classifications. Further evaluation is needed to determine the variant's clinical significance and potential health effects.\nBiotinidase Deficiency\nDescription: Biotinidase deficiency is a rare genetic disorder caused by the inability to recycle biotin, a vitamin necessary for energy production and maintenance of healthy skin, hair, and mucous membranes. This condition leads to a buildup of biotin in the bloodstream, which eventually depletes due to impaired recycling mechanisms. Symptoms include neurological issues such as seizures, developmental delays, muscle weakness, and dermatitis. If left untreated, severe cases can lead to life-threatening complications, including coma or death. Early detection through newborn screening is crucial for effective management with biotin supplements.\nYeo CJJ, Tizzano EF, Darras BT. Challenges and opportunities in spinal muscular atrophy therapeutics. The Lancet. Neurology. 2024 Feb. PMID: 38267192\nAdam MP, Feldman J, Mirzaa GM et al.. Biotinidase Deficiency. . 1993. PMID: 20301497\nDodsworth BT, Reeve K, Falco L et al.. Development and validation of an international preoperative risk assessment model for postoperative delirium. Age and ageing. 2023 Jun 1. PMID: 37290122\nSadeghirad B, Dodsworth BT, Schmutz Gelsomino N et al.. Perioperative Factors Associated With Postoperative Delirium in Patients Undergoing Noncardiac Surgery: An Individual Patient Data Meta-Analysis. JAMA network open. 2023 Oct 2. PMID: 37819663",
    "Gene Name": "BTD",
    "Protein Change": "p.D424H",
    "Coding Change": "c.G1270C",
    "ClinVar Significance": "Pathogenic/Likely_pathogenic",
    "Genomiki Variant Classification": "Uncertain Significance",
    "Disease Associated": "Biotinidase deficiency\n",
    Zygosity: "Heterozygous",
    Chromosome: "chr3",
    Position: "15686693",
    "Ref Allele": "G",
    "Alt Allele": "C",
    "Transcript ID": "NM_001370658",
    "Exon Number": "exon4",
    Depth: "36",
    "gnomAD Allelic Frequency": "0.0285",
    "Gene Effect": "exonic",
    "Mutation Effect": "nonsynonymous_SNV",
    SIFT: "D",
    Polyphen2: "D",
    MutationTaster: "A",
    dbSNPID: "rs13078881",
    "Inheritance Pattern": "Autosomal Recessive",
    "Evidence Code": "PP5,PM5,PM1,PP3,PP2,BS2,BA1",
    OMIM: "253260",
  },
];

export { interpretation1, interpretation2, interpretation3, interpretation4 };
