const interpretationOnquer1 = [
  {
    id: "1",
    Interpretation:
      "The c.2573T>G (p.Leu858Arg) variant in the EGFR gene is a missense mutation, where the amino acid leucine is replaced by arginine at position 858. This change occurs in the tyrosine kinase domain of the EGFR protein, which is important for its role in cell signaling and growth.It is classified as Pathogenic (according to ACMG guidelines) and is Tier 1 (according to AMP guidelines), indicating strong clinical significance, especially for targeted therapy decisions.",
    "Gene Name": "EGFR",
    "Protein Change": "p.Leu858Arg",
    "Coding change": "c.2573T>G",
    "ClinVar Significance": "Pathogenic",
    "Genomiki ACMG": "Pathogenic",
    "Genomiki AMP": "Tier1",
    Depth: "115",
    VAF: "46.5",
    Chromosome: "chr7",
    Position: "55259515",
    "Ref allele": "T",
    "Alt allele": "G",
    "Transcript ID": "NM_005228",
    "Exon Number": "exon21",
    "gnomAD Allelic Frequency": ".",
    "Gene effect": "exonic",
    "Mutation Effect": "nonsynonymous_SNV",
    SIFT: "D",
    Polyphen2: "D",
    MutationTaster: "A",
    "dbSNP ID": "rs121434568",
    cosmic: "ID\\x3dCOSM6224",
    "Allelic Depth Reference": "161",
    "Allelic Depth Alt": "109",
    "Fisher Strand(FS)*": "0",
    "StrandOddsRatio(SOR)**": "0.548",
  },
];

const interpretationOnquer2 = [
  {
    id: "1",
    Interpretation: `The c.2864+1G>A variant in the POLE gene affects a canonical splice site, which is crucial for proper processing of the gene's RNA. Changes at this position can lead to incorrect or absent protein production.
Variants that disrupt normal splicing are often associated with disease, especially when they occur in genes involved in DNA replication and repair, like POLE.
Based on available evidence, this variant is classified as Pathogenic (according to ACMG guidelines) and falls under Tier 2 (according to AMP guidelines), indicating it has significant clinical relevance, particularly in the context of hereditary cancer predisposition.`,
    "Gene Name": "POLE",
    "Protein Change": ".",
    "Coding change": "c.2864+1G>A",
    "ClinVar Significance": "Likely_pathogenic",
    "Genomiki ACMG": "Pathogenic",
    "Genomiki AMP": "Tier2",
    Depth: "128",
    VAF: "17.64705882",
    Chromosome: "chr12",
    Position: "133238112",
    "Ref allele": "C",
    "Alt allele": "T",
    "Transcript ID": "NM_006231",
    "Exon Number": "exon24",
    "gnomAD Allelic Frequency": ".",
    "Gene effect": "splicing",
    "Mutation Effect": ".",
    SIFT: ".",
    Polyphen2: ".",
    MutationTaster: "D",
    "dbSNP ID": ".",
    cosmic: "ID\\x3dCOSV106474230\\x3bOCCURENCE\\x3d1(COSO29206078)",
    "Allelic Depth Reference": "98",
    "Allelic Depth Alt": "21",
    "Fisher Strand(FS)*": "30.672",
    "StrandOddsRatio(SOR)**": "4.774",
  },
  {
    id: "2",
    Interpretation: `The c.2542G>T (p.Glu848Ter) variant in the BRCA1 gene is a nonsense mutation, which introduces a premature stop codon at amino acid position 848. This change is expected to result in a truncated, non-functional protein or lead to nonsense-mediated mRNA decay.
Such loss-of-function variants in BRCA1, a gene involved in DNA repair, are often associated with an increased risk of hereditary breast and ovarian cancer.
Based on current evidence, this variant is classified as Likely Pathogenic (according to ACMG guidelines) and falls under Tier 1 (according to AMP guidelines), indicating strong clinical significance.`,
    "Gene Name": "BRCA1",
    "Protein Change": "p.E848X",
    "Coding change": "c.G2542T",
    "ClinVar Significance": ".",
    "Genomiki ACMG": "Likely pathogenic",
    "Genomiki AMP": "Tier1",
    Depth: "101",
    VAF: "93.5483871",
    Chromosome: "chr17",
    Position: "41245006",
    "Ref allele": "C",
    "Alt allele": "A",
    "Transcript ID": "NM_007294",
    "Exon Number": "exon10",
    "gnomAD Allelic Frequency": ".",
    "Gene effect": "exonic",
    "Mutation Effect": "stopgain",
    SIFT: ".",
    Polyphen2: ".",
    MutationTaster: "A",
    "dbSNP ID": ".",
    cosmic: "ID\\x3dCOSV58800087\\x3bOCCURENCE\\x3d1(COSO29914826)",
    "Allelic Depth Reference": "6",
    "Allelic Depth Alt": "87",
    "Fisher Strand(FS)*": "88.821",
    "StrandOddsRatio(SOR)**": "8.955",
  },
];

const interpretationOnquer3 = [
  {
    id: "1",
    Interpretation: `The c.2542G>T (p.Glu848Ter) variant in the BRCA1 gene is a nonsense mutation, which changes a glutamic acid to a stop codon at position 848. This leads to a premature termination of the BRCA1 protein, likely resulting in a loss of normal protein function.
Loss-of-function variants in BRCA1 are associated with an increased risk of hereditary breast and ovarian cancer.
This variant is classified as Likely Pathogenic (according to ACMG guidelines) and belongs to Tier 1 (according to AMP guidelines), indicating strong clinical significance.`,
    "Gene Name": "BRCA1",
    "Protein Change": "p.E848X",
    "Coding change": "c.G2542T",
    "ClinVar Significance": ".",
    "Genomiki ACMG": "Likely pathogenic",
    "Genomiki AMP": "Tier1",
    Depth: "84",
    VAF: "98.80952381",
    Chromosome: "chr17",
    Position: "41245006",
    "Ref allele": "C",
    "Alt allele": "A",
    "Transcript ID": "NM_007294",
    "Exon Number": "exon10",
    "gnomAD Allelic Frequency": ".",
    "Gene effect": "exonic",
    "Mutation Effect": "stopgain",
    SIFT: ".",
    Polyphen2: ".",
    MutationTaster: "A",
    "dbSNP ID": ".",
    cosmic: "ID\\x3dCOSV58800087\\x3bOCCURENCE\\x3d1(COSO29914826)",
    "Allelic Depth Reference": "1",
    "Allelic Depth Alt": "83",
    "Fisher Strand(FS)*": "0",
    "StrandOddsRatio(SOR)**": "7.476",
  },
  {
    id: "2",
    "Gene Name": "TP53",
    Interpretation: `The c.749C>T (p.Pro250Leu) variant in the TP53 gene is a missense mutation, resulting in the substitution of proline with leucine at position 250. This region lies within the DNA-binding domain of the TP53 protein, which is essential for its role in regulating cell growth and preventing tumor formation.
This variant has been reported in association with various cancers and is believed to affect the normal function of TP53, a key tumor suppressor gene. It is classified as Pathogenic (according to ACMG guidelines) and falls under Tier 1 (according to AMP guidelines), indicating high clinical relevance.`,
    "Protein Change": "p.P250L",
    "Coding change": "c.C749T",
    "ClinVar Significance": "Pathogenic/Likely_pathogenic",
    "Genomiki ACMG": "Pathogenic",
    "Genomiki AMP": "Tier1",
    Depth: "232",
    VAF: "80.43478261",
    Chromosome: "chr17",
    Position: "7577532",
    "Ref allele": "G",
    "Alt allele": "A",
    "Transcript ID": "NM_000546",
    "Exon Number": "exon7",
    "gnomAD Allelic Frequency": ".",
    "Gene effect": "exonic",
    "Mutation Effect": "nonsynonymous_SNV",
    SIFT: "D",
    Polyphen2: "D",
    MutationTaster: "D",
    "dbSNP ID": "rs1064794311",
    cosmic:
      "ID\\x3dCOSV52687307\\x3bOCCURENCE\\x3d1(COSO29645555),1(COSO28454826),2(COSO28245257),2(COSO28245232),1(COSO29096562),1(COSO36704830),3(COSO31394830),2(COSO29005278),1(COSO29914830),1(COSO27985184),2(COSO28394862),1(COSO36004826),1(COSO28664826),1(COSO29645579),1(COSO29974826),2(COSO30135468),2(COSO29244826),1(COSO27774843),1(COSO59385343),1(COSO36004862),1(COSO30454862),1(COSO37914808),1(COSO29915333)",
    "Allelic Depth Reference": "36",
    "Allelic Depth Alt": "148",
    "Fisher Strand(FS)*": "1.476",
    "StrandOddsRatio(SOR)**": "0.548",
  },
];

export { interpretationOnquer1, interpretationOnquer2, interpretationOnquer3 };
