const interpretationColumns = [
  { field: "id", headerName: "ID", width: 2 },
  { field: "Gene Name", headerName: "Gene Name", width: 130 },
  { field: "Protein Change ", headerName: "Protein Change", width: 130 },
  { field: "Coding Change", headerName: "Coding Change", width: 130 },
  { field: "ClinVar", headerName: "ClinVar", width: 100 },
  {
    field: "Genomiki Variant Classification",
    headerName: "Genomiki Variant Classification",
    width: 200,
  },
  {
    field: "Inheritance Pattern",
    headerName: "Inheritance Pattern",
    width: 170,
  },
  { field: "Zygosity", headerName: "Zygosity", width: 120 },
  { field: "Disease Associated", headerName: "Disease Associated", width: 200 },
  { field: "Depth", headerName: "Depth", width: 80 },
  {
    field: "gnomAD Allelic Frequency",
    headerName: "gnomAD Allelic Frequency",
    width: 180,
  },
  { field: "dbSNP ID", headerName: "dbSNP ID", width: 120 },
  { field: "Transcript ID", headerName: "Transcript ID", width: 140 },
  { field: "Exon Number", headerName: "Exon Number", width: 120 },
  { field: "Chromosome ", headerName: "Chromosome", width: 110 },
  { field: "Position", headerName: "Position", width: 110 },
  { field: "Ref allele", headerName: "Ref allele", width: 110 },
  { field: "Alt Allele", headerName: "Alt Allele", width: 110 },
  { field: "Gene Effect", headerName: "Gene Effect", width: 120 },
  { field: "Mutation Effect ", headerName: "Mutation Effect", width: 140 },
  { field: "SIFT", headerName: "SIFT", width: 80 },
  { field: "Polyphen2", headerName: "Polyphen2", width: 100 },
  { field: "Mutation Taster", headerName: "Mutation Taster", width: 120 },
  { field: "OMIM", headerName: "OMIM", width: 100 },
  { field: "Evidence Code", headerName: "Evidence Code", width: 140 },
  { field: "Comments", headerName: "Comments", width: 200 },
  { field: "Interpretation", headerName: "Interpretation", width: 800 },
];

const InterpretationData = [
  {
    id: 1,
    "Gene Name": "ATAD3B",
    "Protein Change ": "p.G28R",
    "Coding Change": "c.G82A",
    ClinVar: "-",
    "Genomiki Variant Classification": "Uncertain Significance",
    "Inheritance Pattern": "Autosomal Dominant",
    Zygosity: "Heterozygous",
    "Disease Associated": ".",
    Depth: "45",
    "gnomAD Allelic Frequency": "-",
    "dbSNP ID": "rs766445790",
    "Transcript ID": "-",
    "Exon Number": "-",
    "Chromosome ": "-",
    Position: "-",
    "Ref allele": "-",
    "Alt Allele": "-",
    "Gene Effect": "-",
    "Mutation Effect ": "-",
    SIFT: "-",
    Polyphen2: "-",
    "Mutation Taster": "-",
    OMIM: "-",
    "Evidence Code": "-",
    Comments: "-",
    Interpretation:
      "A variant with accession number VCV002248619 has been identified. This variant is titled 'NM_031921.6(ATAD3B):c.82G>A (p.Gly28Arg)'. It is a single nucleotide variant variant, with a cDNA change of c.82G>A and a protein change of G28R.The variant is located on chromosome 1, spanning from position 1471966 to 1471966. This variant is associated with the trait 'not specified' and has been classified as 'Uncertain significance'. The review status for this variant is 'criteria provided, single submitter', and it was last evaluated on 2021/08/02 00:00. dbSNP link: N/A ClinGen link: N/A ATAD3B encodes a mitochondrial protein involved in regulating cell metabolism, particularly in fatty acid oxidation. A single nucleotide variant, c.82G>A (p.Gly28Arg), has been identified on chromosome 1. Although associated with an unspecified trait, this variant is classified as \\\"\"Uncertain significance\\\"\" and its impact is unclear. Further study is needed to determine its functional effect on ATAD3B protein function. Disease 1. **Leber's Hereditary Optic Neuropathy (LHON)**: A mitochondrial disorder characterized by progressive vision loss. 2. **Mitochondrial Encephalomyopathy, Lactic Acidosis, and Stroke-like episodes (MELAS)**: A mitochondrial disorder that affects energy production in cells. 3. **Sideroblastic Anemia**: A group of disorders characterized by impaired heme synthesis and accumulation of iron within mitochondria. 4. **Pearson Syndrome**: A rare disorder characterized by failure to thrive, lactic acidosis, and sideroblastic anemia. 5. **Mitochondrial Myopathy**: A disorder that affects the muscles, leading to weakness, fatigue, and other symptoms related to impaired mitochondrial function. Please note that these associations are based on the related ATAD3A gene, not directly from the provided text for the ATAD3B gene. Further research is needed to confirm any potential links between the ATAD3B gene and these conditions. REFERENCES: 1. Li S, Rousseau D. ATAD3, a vital membrane bound mitochondrial ATPase involved in tumor progression. Journal of bioenergetics and biomembranes. 2012 Feb. PMID: 22318359 2. Baudier J. ATAD3 proteins: brokers of a mitochondria-endoplasmic reticulum connection in mammalian cells. Biological reviews of the Cambridge Philosophical Society. 2018 May. PMID: 28941010 3. Li S, Rousseau D. [ATAD3, a vital membrane-bound mitochondrial ATPase involved in tumor progression]. Medecine sciences : M/S. 2011 Dec. PMID: 22192748 4. Holt IJ, He J, Mao CC et al.. Mammalian mitochondrial nucleoids: organizing an independently minded genome. Mitochondrion. 2007 Sep. PMID: 17698423 ",
  },
  {
    id: 2,
    "Gene Name": "NOC2L",
    "Protein Change ": "p.S556L",
    "Coding Change": "c.C1667T",
    ClinVar: "-",
    "Genomiki Variant Classification": "Uncertain Significance",
    "Inheritance Pattern": "Autosomal Dominant",
    Zygosity: "Heterozygous.",
    "Disease Associated": "",
    Depth: ".",
    "gnomAD Allelic Frequency": ".",
    "dbSNP ID": "rs35471880",
    "Transcript ID": ".",
    "Exon Number": ".",
    "Chromosome ": ".",
    Position: ".",
    "Ref allele": ".",
    "Alt Allele": "..",
    "Gene Effect": "",
    "Mutation Effect ": ".",
    SIFT: ".",
    Polyphen2: ".",
    "Mutation Taster": ".",
    OMIM: ".",
    "Evidence Code": ".",
    Comments: ".",
    Interpretation:
      "The variant located in the NOC2L gene at protein position p.S556L and coding position c.C1667T is classified as Uncertain Significance. This variant is inherited in a Autosomal Dominant pattern and is present in a Heterozygous state. This variant is associated with . NOC2L is a gene that plays a crucial role in regulating the nuclear organization and transcriptional regulation of genes. Its function involves the binding of histones, which are key components of chromatin structure, to influence gene expression. NOC2L is involved in various cellular processes, including cell proliferation, differentiation, and response to DNA damage. NOC2L is associated with pathways such as histone modification, chromatin remodeling, and transcriptional regulation. It interacts with other proteins and factors to modulate the activity of specific genes. Additionally, NOC2L has been implicated in the regulation of gene expression during cellular development and differentiation.Research on NOC2L has focused on its role in maintaining genome stability and regulating cell growth. Studies have also investigated its potential involvement in various cellular processes, including DNA repair and replication. Further research is needed to fully understand the mechanisms underlying NOC2L's function and regulation. 1. **NOC2L-Related Disorder**: This condition is a rare disorder caused by mutations in the NOC2L gene. It leads to intellectual disability, developmental delay, and other physical abnormalities. 2. **Congenital Disorders of Glycosylation (CDG)**: While not exclusively linked to NOC2L, some forms of CDG have been associated with histone modifications that involve proteins related to or influenced by the NOC2L gene's function in cellular processes such as transcription and DNA replication. 3. **Neurodevelopmental Disorders**: Given NOC2L's role in brain development and its involvement in histone modification, which is crucial for neuronal health and cognitive functions, individuals with mutations affecting this gene might be at risk of neurodevelopmental disorders like autism spectrum disorder (ASD) or attention deficit hyperactivity disorder (ADHD). However, these conditions are complex and multifactorial, and direct causality with NOC2L needs further study. 4. **Epigenetic Disorders**: Given the role of histone modifications in gene expression regulation, mutations affecting genes involved in this process can lead to epigenetic disorders, which affect how cells read their DNA without changing the underlying sequence. Examples include Rett syndrome and other conditions caused by mutations leading to altered gene expression. 5. **Genetic Intellectual Disability**: Intellectual disability can result from a wide range of genetic causes, including mutations affecting histone modification genes. NOC2L's involvement in these processes suggests it could be related to intellectual disabilities caused by histone-related disruptions, though more research is needed to clarify its direct contribution. Please note that the information provided is based on limited data and understanding of the role of NOC2L in disease. Further research is necessary for definitive conclusions about its association with specific diseases or conditions.",
  },
];

export { interpretationColumns, InterpretationData };
